pytorch/buildspec-neuron.yml

```diff
@@ -24,7 +27,7 @@
     BuildNeuronPTInferencePy3DockerImage:
       <<: *INFERENCE_REPOSITORY
       build: &PYTORCH_CPU_INFERENCE_PY3 false
-      image_size_baseline: 4899
+      image_size_baseline: 10000
```

Comment:
What is the true size of the Neuron Inference DLC image?

Reply:
Around 8G now.